pkg/util/ipsec/ipsec_telemetry.go

```diff
@@ -0,0 +1,39 @@
+package ipsec
+
+import (
+	"strconv"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog/v2"
+)
+
+var (
+	ipsecStateGauge *metrics.GaugeVec
+
+	recordedState     string
+	recordedLegacyAPI string
+)
+
+func init() {
+	ipsecStateGauge = metrics.NewGaugeVec(&metrics.GaugeOpts{
```

Comment:
please fill in fields `namespace`, `subsystem` and `name`. Its how the full name is usually constructed in OCP.

Reply:
I see that namespace is hardly used and this metric is cluster level and not really namespaced (in meaning that it is limited to a specific namespace and can differ between namespaces) so I just used only subsystem and name (which is the common way to create metrics as far as I could find)

Reply:
Its cluster level but in order to easily inform the consumer where it came from, we populate the `namespace` field.